charts/elasticsearch/README.md

```diff
@@ -0,0 +1,71 @@
+# elasticsearch
```

Comment:
Even viewing this document as source makes me so happy. Very nicely done.

Reply:
It's actually auto-generated documentation using [helm-docs](https://github.com/norwoodj/helm-docs)